### PR TITLE
Fix the help for the Cautious mode

### PR DESCRIPTION
**Pull request by setton**
When the Cautious mode catches something, print a more helpful
message.

Adapt the Makefile so that it doesn't hard code a temporary directory,
allowing runs by multiple users or concurrent runs by one user.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,2 @@
 test:
-	rm -rf /tmp/_adareducer_test
-	cp -R tests/orig /tmp/_adareducer_test && \
-    gprbuild -P /tmp/_adareducer_test/p.gpr && \
-    python adareducer.py --single-file /tmp/_adareducer_test/proc.adb --follow-closure /tmp/_adareducer_test/p.gpr tests/basic.sh
+	bash tests/test.sh
```
Comment (pmderodat) [inline Makefile]: This hunk and the new file belong to another commit. Also, can you restore the linefeed character at the end of the file?

Comment (setton) [reply Makefile]: Done and done!

**Review by pmderodat — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (pmderodat) [reply Makefile]: Thanks!
